### ISSUE TITLE
yaml multimappers options

### ISSUE THREAD

**Issue by mlizio** [sdparekh/zUMIs#301]
Hi,

Does setting 'primaryHit: yes', the default, mean that featureCounts counts unique alignments as well as the best alignment for reads mapping to multiple locations? 

I am running zUMIs on a mixed human and mouse single cells dataset, and wanted to have a breakdown of what fraction of the identified barcodes correctly labels cells from one genome only, compared to barcodes labelling mixed genomes.

In other words, how do I distinguish the case of a read that maps equally well to both human and mouse from the case of a read that maps to only one genome? Would I have to set 'primaryHit: no' to avoid multimappers altogether?

thanks

**Comment by mlizio** [comment]
I did solve this by trying both ways and comparing log files and results. Closing the issue.
